tests/services-nextdns.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  nextdns = pkgs.runCommand "nextdns-0.0.0" {} "mkdir $out";
+in
+
+{
+  services.nextdns.enable = true;
+  services.nextdns.arguments = ["-config" "10.0.3.0/24=abcdef"];
+
+  test = ''
+    echo >&2 "checking nextdns service in ~/Library/LaunchAgents"
+    grep "org.nixos.nextdns" ${config.out}/user/Library/LaunchAgents/org.nixos.nextdns.plist
```

```suggestion
    grep "org.nixos.nextdns" ${config.out}/Library/LaunchDaemons/org.nixos.nextdns.plist
```

Since this is a demon and not an agent.

---

Thanks!